### ISSUE TITLE
[Feature Request] - include mp4 link in the file name or text dump

### ISSUE THREAD

**Issue by epicsoulzero** [tharyckgusmao/Skill-Share-Crawler---DL#37]
hi im just new here, im just wondering if you could add a feature to add the actual video file link (brightcove link) on the file name being downloaded.  or something like a dump text with the links. thanks

**Comment by Darwish8040** [comment]
Yes, please! This is a great feature, I can use this to download with my download manager.
